python/GangaLHCb/__init__.py

```diff
@@ -82,16 +83,30 @@ def _store_root_version():
 
 def _store_dirac_environment():
     from GangaDirac.Lib.Utilities.DiracUtilities import write_env_cache, get_env
-    diracversion = _guess_version('LHCBDIRAC')
```

Comment:
You can also remove the `_guess_version` code completely with this PR as this is the only use of it.